app/src/main/res/layout/suggestions_strip.xml

```diff
@@ -52,6 +52,7 @@
             android:layout_height="match_parent"
             android:layout_weight="1"
             android:hapticFeedbackEnabled="false"
+            android:gravity="center_horizontal"
```

Comment:
What is the purpose / effect of this?

Reply:
This change of gravity is to ensure that a clipboard suggestion (which is not wide enough to fill the entire suggestion strip view) will be centered as opposed to aligned to the left (or to the right if an RTL subtype is selected)